### PR TITLE
Explicitly require setuptools, towncrier/_settings.py imports pkg_resources

### PR DESCRIPTION
**Pull request by hroncok**

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -41,7 +41,7 @@
         "Programming Language :: Python :: Implementation :: PyPy",
     ],
     use_incremental=True,
-    install_requires=["click", "click-default-group", "incremental", "jinja2", "toml"],
+    install_requires=["click", "click-default-group", "incremental", "jinja2", "toml", "setuptools"],
```
Comment (altendky) [inline setup.py:44]:
```suggestion
    install_requires=["click", "click-default-group", "incremental", "jinja2", "setuptools", "toml"],
```

**Review by altendky — CHANGES_REQUESTED** [review]
Hi @hroncok, thanks for noticing this and submitting a correction.  I've got a couple nitpicks.  The alphabetical ordering of includes and a blank `src/towncrier/newsfragments/333.misc.rst` needs to be added.  If you could address those two points then we can merge this.  Or let me know and I'll take care of the boilerplate.  I just try not to encroach on other people's PRs without their approval.

**Comment by hroncok** [comment]
Thanks for offering to deal with the bolierplate. I appreciate it!

I've clicked trough the GitHub UI to make ti happen. Feel free to squash (or to do absolutely any other changes in the PR in case I've made something in a way that is not ideal).

**Comment by altendky** [comment]
Looks good, thanks.  I'll let the CI finish out just...  well, that's what CI does.  `:]`  Then merge time.

**Pull request merged**